File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int ans[] = new int[2*nums.length];
        for(int i = 0;i<2*nums.length;i++){
            int counter = i%nums.length;
            ans[i] = nums[counter];
        }
        return ans;
    }
}

File: Data Structures & Algorithms/is-anagram/submission-0.py
class Solution(object):
    def isAnagram(self, s, t):
        if len(s)!=len(t):
            return False
        ans = dict()
        for i in s:
            ans[i] = ans.get(i,0)+1
        for i in t:
            if i not in ans or ans[i]==0:
                return False
            ans[i]-=1
        return True